dashboard/Dockerfile

```diff
@@ -20,9 +20,9 @@ RUN apt-get update && \
     && rm -rf /var/lib/apt/lists/*
 COPY package.json /dashboard/
 COPY yarn.lock /dashboard/
-RUN cd /dashboard && npm i yarn && npx yarn install --ignore-scripts
+RUN cd /dashboard && npm i yarn && yarn install
```

Comment:
```suggestion
RUN cd /dashboard && npm config set package-lock false && npm i yarn && yarn install
```

Reply:
@qurben This will be stop to generate `package-lock.json`. Does this have any side-effects?

Reply:
@monaka I updated my PR, IMHO not installing yarn at all is much cleaner solution without any side-effects. I would just recommend to update the base image from `node:8.10.0` to `node:8.15.1` which contains latest yarn